packages/fluent-theme/src/fluent/styles/ComboBox.styles.ts

```diff
@@ -1,13 +1,61 @@
 import { fluentBorderRadius } from './styleConstants';
 import { Depths } from '../FluentDepths';
+import { NeutralColors, SharedColors } from '../FluentColors';
 
 export const ComboBoxStyles = {
   root: {
-    borderRadius: fluentBorderRadius // the bound input box
+    borderRadius: fluentBorderRadius, // the bound input box
+    borderColor: NeutralColors.gray80,
+    paddingLeft: 8
+  },
+  rootHovered: {
+    borderColor: NeutralColors.gray160,
+    selectors: {
+      '.ms-ComboBox-Input': {
+        color: NeutralColors.gray190
+      }
+    }
+  },
+  rootError: {
+    borderColor: SharedColors.red10 // current structure of ComboBox does not allow to change the hover/focus color when has error
   },
   callout: {
-    borderRadius: `0 0 ${fluentBorderRadius} ${fluentBorderRadius}`,
-    overflow: 'hidden',
-    boxShadow: Depths.depth8
+    borderRadius: fluentBorderRadius,
+    border: 'none',
+    boxShadow: Depths.depth8,
+    selectors: {
+      ['.ms-Callout-main']: { borderRadius: fluentBorderRadius }
```

Comment:
Don't think I've seen selectors in an array like this before. Is that intentional?

Reply:
This worked for dropdown and I thought of not trying a different approach, even though I think it will work without the []. Will push a change after testing it.